<ui version="4.0" >
 <class>panelWidget</class>
 <widget class="QWidget" name="panelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>347</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>3000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Pyranha control panel</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Theories path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="theoriesPathLineEdit" />
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="theoriesPathButton" >
         <property name="minimumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Select path for theories of motion</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="__resources.qrc" >:/images/folder_open.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QPushButton" name="resetPathButton" >
         <property name="minimumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Reset path for theories of motion to default value</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="__resources.qrc" >:/images/reload.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>I/O format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3" >
        <widget class="QComboBox" name="comboBox" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <item>
          <property name="text" >
           <string>Plain</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Latex</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Floating point representation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QComboBox" name="fpComboBox" >
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Number of decimal digits</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QSlider" name="digitsSlider" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2" >
        <widget class="QLabel" name="digitsLabel" >
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>129</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2" >
      <attribute name="title" >
       <string>Psymbols</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="latexRenderCheckBox" >
         <property name="toolTip" >
          <string>Enable TeX rendering (a TeX installation must be available)</string>
         </property>
         <property name="text" >
          <string>LaTeX rendering</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QProgressBar" name="latexRenderingProgressBar" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>150</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="format" >
          <string>Rendering - %p%</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QTreeWidget" name="treeWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="rootIsDecorated" >
          <bool>true</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="animated" >
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radioButton" >
            <property name="text" >
             <string>RadioButton</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="radioButton_2" >
            <property name="text" >
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>183</width>
           <height>311</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="__resources.qrc" />
 </resources>
 <connections/>
</ui>
